tests/cases/callbacks/init.slint

```diff
@@ -61,7 +61,9 @@ TestCase := Rectangle {
     }
 
     Container {
-
+        init => {
+            InitOrder.observed-order += "|container-instantiation";
```

Comment:
Would be nice if this (and the other) would call local properties to be sure that the property access works despite inlining. (I'm sure it does but it would be nice to test it)

Reply:
Good idea, I'll adapt the test to take this string value from a local property.

Reply:
Good thing that you brought it up. It seems to work, but I've found one case where it doesn't:

```
thread 'main' panicked at internal/compiler/namedreference.rs:167:28:
already mutably borrowed: BorrowError
```

I'll have a look.